arches_resource_sets/models.py

```diff
@@ -1,26 +1,54 @@
 import uuid
-from django.db import models
+from django.db import IntegrityError, models
 from django.conf import settings
 
 from arches.app.models.models import ResourceInstance
 from arches.app.models.models import I18n_TextField
 
 
 class ResourceSet(models.Model):
-    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, serialize=True)
```

Comment:
I don't think "serialize" is actually a valid property of a model.Field, we should probably just remove

Reply:
Ah yes, Jacob did some legwork on that and found it missing.  Just comes from arches models.py - you can see it in there all over the place.  Probably should be purged at some point.